### PR TITLE
srfcms-6780-video-gallery-the-2nd

### PR DESCRIPTION
**Pull request by nobelreg**

### CODE REVIEW

**Comment by rothfahl** [comment]
LGTM

File: source/_patterns/10-atoms/content/media-window.twig [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<div class="media-window l-full-width-container">
+    <button class="media-window__close" value="{{ 'Schliessen'|trans }}"></button>
```
Comment (pchr-srf) [inline source/_patterns/10-atoms/content/media-window.twig]: This breaks the build :(

Comment (pchr-srf) [inline source/_patterns/10-atoms/content/media-window.twig]: Fixed: https://github.com/mmz-srf/srf-frontend-framework/commit/6c4635ffc28ce0a013848f55e8b5d42fb6124cee

Comment (rothfahl) [inline source/_patterns/10-atoms/content/media-window.twig]: danke @pchr-srf!

**Pull request merged**